src/css/custom.css

```diff
@@ -377,48 +388,56 @@ nav.menu {
   top: 44px;
   border-radius: 0;
   box-shadow: none;
-  height: 296px;
+  height: var(--ifm-search-dropdown-height);
   max-width: none;
-  background: white;
+  background: var(--ifm-color-brand-light);
 }
 
-.algolia-autocomplete .ds-dropdown-menu [class^=ds-dataset-] {
+.algolia-autocomplete .ds-dropdown-menu [class^="ds-dataset-"] {
   border-radius: 0 !important;
 }
 
 .grid-bg-section {
-  background-image: linear-gradient(var(--ifm-color-black-opacity-05) 1px, transparent 1px),
-    linear-gradient(90deg, var(--ifm-color-black-opacity-05) 1px, transparent 1px);
-  background-size: 52px 52px;
+  background-image: linear-gradient(var(--ifm-color-brand-dark-opacity-05) 1px, transparent 1px),
+    linear-gradient(90deg, var(--ifm-color-brand-dark-opacity-05) 1px, transparent 1px);
+  background-size: var(--ifm-grid-bg-size-lg) var(--ifm-grid-bg-size-lg);
   clip-path: inset(0.5rem 0 1rem 0);
 }
 
 .button-grid-bg-section {
-  background-image: linear-gradient(var(--ifm-color-white-opacity-15) 1px, transparent 2px),
-    linear-gradient(90deg, var(--ifm-color-white-opacity-15) 1px, transparent 2px);
-  background-size: 20px 20px;
+  background-image: linear-gradient(var(--ifm-color-brand-light-opacity-15) 1px, transparent 2px),
+    linear-gradient(90deg, var(--ifm-color-brand-light-opacity-15) 1px, transparent 2px);
+  background-size: var(--ifm-grid-bg-size-sm) var(--ifm-grid-bg-size-sm);
   clip-path: inset(0.2rem);
   position: absolute;
   border-radius: 20px;
 }
 
 .button-grid-bg-section-dark {
-  background-image: linear-gradient(var(--ifm-color-black-opacity-05) 1px, transparent 2px),
-    linear-gradient(90deg, var(--ifm-color-black-opacity-05) 1px, transparent 2px);
-  background-size: 20px 20px;
+  background-image: linear-gradient(var(--ifm-color-brand-dark-opacity-05) 1px, transparent 2px),
+    linear-gradient(90deg, var(--ifm-color-brand-dark-opacity-05) 1px, transparent 2px);
+  background-size: var(--ifm-grid-bg-size-sm) var(--ifm-grid-bg-size-sm);
   position: absolute;
   border-radius: 20px;
 }
 
 .button-grid-bg-github {
-  background-image: linear-gradient(var(--ifm-color-black-opacity-05) 1px, transparent 2px),
-    linear-gradient(90deg, var(--ifm-color-black-opacity-05) 1px, transparent 2px);
-  background-size: 20px 20px;
+  background-image: linear-gradient(var(--ifm-color-brand-dark-opacity-05) 1px, transparent 2px),
+    linear-gradient(90deg, var(--ifm-color-brand-dark-opacity-05) 1px, transparent 2px);
+  background-size: var(--ifm-grid-bg-size-sm) var(--ifm-grid-bg-size-sm);
+  scale: 0.9;
+  opacity: 0;
+}
+
+.button-grid-bg-github:hover {
+  scale: 1;
+  opacity: 1;
+  transition: all 250ms ease-out;
```

Comment:
shouldn't this be common for all buttons?

Reply:
The button used in navbar is a separate component and the other buttons are separate component so these styles are here.